### ISSUE TITLE
Disable system cursor while using mouse on android

### ISSUE THREAD

**Issue by Schoolmist** [alexbatalov/fallout2-ce#345]
When I plug my mouse to my android tablet it seems to work with fallout 2 CE but unfortunately I can also see the system cursor. Is there any way to fix this?

https://github.com/alexbatalov/fallout2-ce/assets/145881173/568cd837-6fb0-481e-b0d2-8dc7fb57b588

**Comment by Whateverman81** [comment]
Yeah, i have the same problem on iOS. Would be nice to disable the system cursor. I mean its playable but it is a little bit distracting.
